scss/core/_utilities.scss

```diff
@@ -18,6 +18,13 @@ $color-levels: 100, 200, 300, 400, 500, 600, 700, 800, 900;
   font-size: $x-small-font-size;
 }
 
+.disclaimer {
+  font-size: $disclaimer-font-size;
+  font-weight: normal;
+  line-height: $disclaimer-line-height;
+  color: $gray-500;
+}
```

Comment:
I've spoken to the design team about the naming of this. Let's not use the semantic name "disclaimer" here and instead simply add an `xx-small` size.
```
.xx-small {
  font-size: $xxs-font-size;
  line-height: $xxs-line-height;
}
```

And let's keep the color out of Paragon in the app for now:
```
<p className="xx-small text-muted">Legal disclaimer...</p>